src/DataIntegrity.php

```diff
@@ -105,6 +108,38 @@ public static function ensureFieldsPresent(dict<string, mixed> $row, table_schem
               } else {
                 $row[$field_name] = (int)$row[$field_name];
               }
+            } else if ($row[$field_name] is int) {
```

Comment:
I think this can just be `else` since this condition can only be true in this context, due to the prior else if